core/cas-server-core-configuration-api/src/main/java/org/apereo/cas/configuration/DefaultCasConfigurationPropertiesSourceLocator.java

```diff
@@ -168,6 +175,23 @@
         return composite;
     }
 
+    /**
+     * Returns a system property composed of environment variables and system properties.
```

Comment:
You're not returning a system property; You're returning a property source.

Reply:
I just changed the javadoc to:

```
     * Returns a property source composed of system properties and environment variables.
     *
     * System properties take precedence over environment variables (similarly to spring boot behaviour).
```

Let me know if it's ok to you